src/Amount.js

```diff
@@ -28,6 +31,9 @@ class Amount extends Component {
         <button type="button" onClick={this.onDecrement}>
           -
         </button>
+
+        <Euro amount={this.state.amount} />
+        <Pound amount={this.state.amount} />
```

Comment:
Mike1, I was hoping to use the Amount component next week without the currency components.  Let's talk about how we can refactor to make this more reusable.